deploy-teuthology

```diff
@@ -173,19 +173,17 @@ fi
 
 if $SETUPNS ; then
 ZONE_DOMAIN=${OS_CLOUD}.suse.de
-DNSKEY_NAME=$(make_dns_key $ZONE_DOMAIN $PWD/secrets)
-PUBKEY_NAME=${DNSKEY_NAME}.key
-PRIKEY_NAME=${DNSKEY_NAME}.private
-ZONE_SECRET=$(cat $PWD/secrets/$PRIKEY_NAME | yj | jq -r .Key)
+# dnssec key generation was temporarily moved to 'teuthology_nameserver' ansible role due to encryption algorithm deprication on Tumbleweed
+#DNSKEY_NAME=$(make_dns_key $ZONE_DOMAIN $PWD/secrets)
+#PUBKEY_NAME=${DNSKEY_NAME}.key
+#PRIKEY_NAME=${DNSKEY_NAME}.private
+#ZONE_SECRET=$(cat $PWD/secrets/$PRIKEY_NAME | yj | jq -r .Key)
```

Comment:
If it is not used anymore, just delete it

Reply:
and the function make_dns_key is not required anymore

Reply:
Since the algorithm will be deprecated everywhere in the future maybe we will go back to creating the key in the script so that's why I didn't remove it.

Reply:
no... there is no way back, please go ahead clean it up